release-team/role-handbooks/branch-manager/README.md

```diff
@@ -115,6 +115,25 @@ As the first build from the new release branch, publication this build does not
 Adding the --rc flag switches behavior on to building release candidates.  Again the gcbmgr/anago scripting automatically finds and increments the current build number.
 * `./gcbmgr stage release-1.12 --rc --build-at-head --nomock`
 
+In an perfect world, `rc.1` and the official release are the same commit. To get as close to that as we can the following thing should be considered:
+
+1. Everything we have in the milestone needs to be merged before we start staging the release candidate
+
+   [`is:pr is:open milestone:v1.14`][pr-milestone-query] should be empty, so we
+   don't need to cherry-pick those PRs after code thaw
+
+   This means you (the release team) should really push on PRs towards the end
+   of code freeze or assess if they can be removed from the milestone and can
+   be punted to a next release.
+
+2. Code Thaw and cutting the release candidate should coincde
```

Comment:
I think this should be more clear on ordering and on rationale, while also staying abstract and pragmatic.  Is the message intended to be:

1. the rc should not be cut until [`is:pr is:open milestone:v1.14`][pr-milestone-query] is empty (or as very nearly empty as practically possible)
1. code thaw should not happen until the rc has been cut
1. ...because doing otherwise might make it more difficult to get exactly the desired commit into master (conflicting with large tide of thawing merges) and then cherry-picked and at this point time is off the essence to getting any last minute fixes in and validated through CI